pkg/cmd/roachtest/tests/restore.go

```diff
@@ -940,6 +957,38 @@ func (sp restoreSpecs) run(ctx context.Context, c cluster.Cluster) error {
 	return nil
 }
 
+// recordPerf is a rediculous hack to export a single perf metric for the given test to roachperf.
```

Comment:
I feel your frustration :p but the comment isn't adding much, consider deleting it or adding an actionable TODO.